helpers/controllers/userInfoController.ts

```diff
@@ -35,11 +35,19 @@ export const userInfo = async (_parent: void, args: Username) => {
     Star.findAll({
       where: {
         mentorId: user.id
-      }
+      },
+      include: [
+        { model: User, as: 'student', attributes: ['username', 'id', 'name'] },
+        { model: Lesson, as: 'lesson', attributes: ['id', 'title', 'order'] }
+      ]
     })
   ])
   const starMap = starsReceived.reduce((map: any, star: any) => {
     map[star.lessonId] = map[star.lessonId] || []
+    star.dataValues.studentUsername = _.get(star, 'student.username', '')
+    star.dataValues.studentName = _.get(star, 'student.name', '')
+    star.dataValues.lessonOrder = _.get(star, 'lesson.order', 0) // lessonOrder: Int
+    star.dataValues.lessonTitle = _.get(star, 'lesson.title', '')
     map[star.lessonId].push(star.dataValues)
```

Comment:
I sent PR #596 to replace Sequelize on this file, could you wait until that PR is merged and then use Prisma for your code? Sorry about that, but this way we don't add any more tech debt to be fixed later. 

Just FYI, the lesson join for a star won't work on Prisma until we change it to be a foreign key on the table. But the stars are already grouped by lesson in this `starMap` object and also on the `lessonStatus` value on the response. So having another table join here for the lesson is not needed in my opinion. The front-end component already does some work to join the lessons data in order to display the progress from the `lessonStatus` list.

In the future I think this entire query should go through some refactoring. There's a lot of values that go in the response that are not used and just add to the size. For example, the code for each submission from the user is in the response, yet it serves no purpose right now. Also the work that the front-end could be moved into the server response. But that's a thing for another PRs.

Reply:
yup, that works.